generator/internal/dart/templates/lib/field.mustache

```diff
@@ -0,0 +1,20 @@
+{{!
+Copyright 2025 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+}}
+
+{{#Codec.DocLines}}
+/// {{{.}}}
+{{/Codec.DocLines}}
+final {{{Codec.Type}}}? {{Codec.Name}};
```

Comment:
I think this makes all fields nullable (aka optional), right?  In proto3 primitive types (such as `int32` and `string`) are non-nullable with a default value. Only fields marked `optional` are, well, optional.  

I don't know what is the expectation in the Dart community about such things, but maybe we should use a different mapping depending on whether the `Optional` field is set?